bundles/org.openhab.binding.logreader/.classpath

```diff
@@ -6,12 +6,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" output="target/classes" path="src/3rdparty/java">
-		<attributes>
-			<attribute name="optional" value="true"/>
-			<attribute name="maven.pomderived" value="true"/>
-		</attributes>
-	</classpathentry>
```

Comment:
not needed?

Reply:
I fixed the logreader in the new commit https://github.com/openhab/openhab-addons/pull/7270/commits/b529604307c1a3016a8132cade3ba6f987842299